src/client/app/shared/dialog/dialog.component.css

```diff
@@ -0,0 +1,176 @@
+.fl {
```

Comment:
please remove this file , dont maintain any explicit , because our project is in scss no dont maintain sny style sheets like this.